Files/Dialogs/ElevateConfirmDialog.xaml

```diff
@@ -0,0 +1,17 @@
+﻿<ContentDialog
```

Comment:
@yaichenbaum You could use DynamicDialog here :eyes:

Reply:
It'd be easier to maintain

Reply:
@d2dyno1 We can do that in another PR, we need to get this merged in order to ship Files in the Microsoft Store.

Reply:
@yaichenbaum 
```csharp
DynamicDialog dynamicDialog = new DynamicDialog(new DynamicDialogViewModel()
{
            TitleText = "ElevateConfirmDialog/Title".GetLocalized(),
            PrimaryButtonText = "ElevateConfirmDialog/PrimaryButtonText".GetLocalized(),
            CloseButtonText = "ElevateConfirmDialog/CloseButtonText".GetLocalized(),
            DynamicButtons = DynamicDialogButtons.Primary | DynamicDialogButtons.Cancel
});
```

Reply:
@d2dyno1 Does this still make sense to use even though the dialog is going to be used in multiple places?